operator/apis/config/v1/projectconfig_types.go

```diff
@@ -100,8 +100,13 @@ type FeatureGates struct {
 
 	// RuntimeSeccompProfile enables the restricted seccomp profile on all
 	// Lokistack components.
+	// Deprecated: Please use RestrictedPodSecurityStandard instead.
 	RuntimeSeccompProfile bool `json:"runtimeSeccompProfile,omitempty"`
```

Comment:
Since we are owning this configuration exclusively and maintain control over all available bundles, I suggest to delete this feature gate if not used anymore.